compat/history-stream.js

```diff
@@ -62,11 +63,13 @@ exports.init = function (sbot, config) {
             sbot.db
               .getJITDB()
               .paginate(query, 0, limit, false, false, (err, answer) => {
-                cb(err, pull.values(answer.results.map(formatMsg)))
+                if (err) cb(clarify(err, 'ssb-db2 createHistoryStream failed')) // prettier-ignore
```

Comment:
This one is exposed over the network. Can that cause trouble that we might leak information about the local system? I checked compat/ebt and that is fine as getvectorclock is only used internally.